torch/distributed/rpc/_testing/faulty_agent_backend_registry.py

```diff
@@ -62,7 +64,7 @@ def _faulty_process_group_init_backend_handler(
             name,
             group,
             rpc_backend_options.num_send_recv_threads,
-            rpc_backend_options.rpc_timeout,
+            timedelta(seconds=rpc_backend_options.rpc_timeout),
```

Comment:
Do we plan to make this take a float too in subsequent PRs?

Reply:
@mrshenli I decided to not change the internal representations in the RPC agents, the main purpose was to make sure that from the user perspective they pass in float whenever constructing a `RpcBackendOptions` or calling the set/get default timeout APIs. So, the change to float is only planned to be in RpcBackendOptions structs and the relevant APIs.